src/components/ChatGhostUI/ContentLoadingElements/ContentLoadingTwoLines/index.native.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ContentLoader, {Rect, Circle} from 'react-content-loader/native';
```

Comment:
We don't need native variant for all these types of components just import SVG elements from `react-native-svg`.
```js
import {Rect, Circle} from 'react-native-svg';
```

And then it will work on all platforms.

Reply:
Good point. The issue `ContentLoader` is imported from `react-content-loader` in web and `react-content-loader/native` in native.  Should then create a wrapper around `ContentLoader` and then use?

Reply:
Hmm. Let me check

Reply:
I gues that would be better.